exhaustive.go

```diff
@@ -114,11 +114,18 @@ All of these flags are optional.
 
     flag                            type    default value
 
+    -explicit-exhaustive-switch     bool    false
     -check-generated                bool    false
     -default-signifies-exhaustive   bool    false
     -ignore-enum-members            string  (none)
     -package-scope-only             bool    false
 
+
+If the -explicit-exhaustive-switch flag is enabled, the analyzer runs on every
+enum switch statement not explicitly marked with the comment text
+("exhausive:ignore"). Otherwise, it only runs on switch statements marked with
+the comment text ("exhaustive:enforce").
```

Comment:
Shouldn't this be the other way around? Something like:

> If the `-explicit-exhaustive-switch` flag is enabled, then the analyzer only checks switch statements marked with the comment text "exhaustive:enforce". Otherwise, ...

Reply:
Yep, sorry about that. I knew docs would be the main area things would go wrong.